src/content/reference/react-dom/hydrate.md

```diff
@@ -98,23 +98,23 @@ hydrate(<App />, document.getElementById('root'));
 
 ```js App.js
 export default function App() {
-  return <h1>Hello, world!</h1>;
+  return <h1>Halo, dunia!</h1>;
 }
 ```
 
 </Sandpack>
 
-Usually you shouldn't need to call `hydrate` again or to call it in more places. From this point on, React will be managing the DOM of your application. To update the UI, your components will [use state.](/reference/react/useState)
+Biasanya anda tidak perlu memanggil `hydrate` lagi atau memanggilnya di tempat lain. Dari titik ini, React akan mengelola DOM dari aplikasi anda. Untuk memperbarui UI, komponen akan [use state.](/reference/react/useState)
```

Comment:
```suggestion
Biasanya anda tidak perlu memanggil `hydrate` lagi atau memanggilnya di tempat lain. Dari titik ini, React akan mengelola DOM dari aplikasi anda. Untuk memperbarui UI, komponen akan [menggunakan state.](/reference/react/useState)
```

"use state" here is used as a sentence, not a part of the `useState` syntax. So it's more natural to translate it into Bahasa Indonesia.